client/controllers/userEvent.coffee

```diff
@@ -14,6 +14,26 @@ Template.userEvent.onRendered ->
 Template.userEvent.helpers
   isEditing: ->
     return Template.instance().editState.get()
+  incidentLocations: ->
+    locations = {}
+    counts = Template.instance().data.counts
+    # Loop 1: Incident Reports ("counts")
+    for count in counts
+      # Loop 2: Locations within each "count" record
+      for loc in count.locations
```

Comment:
There was a bug in this helper, the fixes are available back in the master branch, file locationList.coffee
Namely, there has to be one more line to check if that `count` has any `.locations` has and `date.parse` should be `Date.parse`